src/dusb/ti84series.js

```diff
@@ -83,6 +83,23 @@ module.exports = class Ti84series {
       flash: params.find(p => p.type == v.parameters.DUSB_PID_FREE_FLASH).value,
     };
   }
+  
+  async delete(name, type) {
+    await this._d.send({
+      type: v.virtualPacketTypes.DUSB_VPKT_MODIF_VAR,
+      data: [
+        0, name.length,
+        ...b.asciiToBytes(name),
+        ...b.constructParameters([{
+          type: v.attributes.DUSB_AID_VAR_TYPE2,
+          value: b.intToBytes(0xF0070000 + type, 4)
```

Comment:
No bounds check on type. Sure, other parts are already doing it wrong too, but no need to introduce more low-quality code :)

Reply:
I'm not sure that type should be a user-supplied value in the first place. How am I, as a user of `ticalc-usb`, supposed to know the possible values for this parameter? :) I'd prefer if we could abstract this knowledge away.

Ideas:
* Auto-detect the type (do a `getDirectory`, then find by name). With the downside that we may delete the wrong thing, or error when there are two options.
* Supply a map of types for the user to choose from (`ticalc.variableTypes.program`) that we can also check against
* Let the user supply a string from a set of valid strings (`'program'`)

Reply:
The type that this accepts is the same that `getDirectory` returns, and the same value as is stored in the entry. Automatic type detection is in my opinion a bad idea, since it might result in code that seems to behave most of the time but then does something completely different the rest of the time. I think an enumeration of types would be the best solution overall.

Reply:
An enumeration of types would work, but it would be a large enum, since types are model-dependent. libtifiles splits types across multiple types*.h files.

Reply:
Really? Why is that? I assume because something like `program` equates to different numeric values depending on the model? If that's the case then again we can solve that issue from the perspective of the API user, because we know the model and we can do the mapping in the library.

Reply:
That, and also that different models have different types available in the first place.

Reply:
Check. Then I'd expect the API to allow the superset of types, convert the right ones and give an error if we try to use a type that isn't valid for the current model. Right..? :)